pages/k8s/ipv6.md

```diff
@@ -0,0 +1,156 @@
+---
+wrapper_template: "kubernetes/docs/base_docs.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "IPv6"
+  description: Using IPv6 with Charmed Kubernetes.
+keywords: juju, network, networking
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: ipv6.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+As of Kubernetes 1.19, support for IPv6 is in beta and dual-stack is in alpha.
+Charmed Kubernetes can enable these by including IPv6 CIDRs in the `cidr`
+config for the Calico charm and the `service-cidr` config for the Kubernetes
+Master charm.  These can contain up to two comma-separated values, with the
+first CIDR in the list being the preferred family for pods, or the default for
+services.  You can override the `IPFamily` for a service when creating it.
+
+<div class="p-notification--positive"><p markdown="1" class="p-notification__response">
+<span class="p-notification__status">Note:</span>
+Calico is the only CNI which supports IPv6 at this time.
+</p></div>
+
+## Example
+
+You can use the following overlay file ([download it here](asset-ipv4-ipv6-overlay])
+to enable IPv4 preferred dual-stack to the [kubernetes-calico][] bundle:
+
+```yaml
+description: Charmed Kubernetes overlay to enable IPv4-IPv6 dual-stack.
+applications:
+  calico:
+    options:
+      cidr: "192.168.0.0/16,fd00:c00b:1::/112"
+  kubernetes-master:
+    options:
+      service-cidr: "10.152.183.0/24,fd00:c00b:2::/112"
+  kubernetes-worker:
+    resources: null
+```
+
+Once that is deployed, you can use the following spec ([download it
+here](asset-nginx-dual-stack)) to run a dual-stack enabled nginx pod with an
```

Comment:
Missing link? Also should be brackets, not parens, unless you want `asset-nginx-dual-stack` to be a literal URL.